test/unit/notebooks/test_validate_notebooks.py

```diff
@@ -27,6 +27,7 @@ def test_no_extra_notebooks(self):
             f'{NOTEBOOK_BASE_DIR}/02-Visualization/Blog Workbench Visualization.ipynb',
             f'{NOTEBOOK_BASE_DIR}/02-Visualization/EPL-Gremlin.ipynb',
             f'{NOTEBOOK_BASE_DIR}/02-Visualization/EPL-SPARQL.ipynb',
+            f'{NOTEBOOK_BASE_DIR}/Introduction-to-Identity-Graphs.ipynb',
             f'{NOTEBOOK_BASE_DIR}/Introduction-to-Knowledge-Graphs.ipynb',
             f'{NOTEBOOK_BASE_DIR}/Introduction-to-Fraud-Graphs.ipynb',
```

Comment:
These should be in a different directory right? Or are we wanting them to be at the top level?

Reply:
We want them to be at the top level @austinkline